### PR TITLE
routes

### PR DESCRIPTION
**Pull request by gbanares**

### CODE REVIEW

File: controllers/childrenController.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  updateChildBudget: function(req, res) {
+    db.Users
+      .findOneAndUpdate({ email: req.params.email}, {budget: req.params.budget})
+      .then(dbModel => res.json(dbModel))
+      .catch(err => res.status(422).json(err));
+  },
+  removeChild: function(req, res) {
+    db.Users
+      .destroy({ email: req.params.email })
```
Comment (jmckenney) [inline controllers/childrenController.js]: Destroying children is not the best connotation within a project:)

File: controllers/childrenController.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const db = require("../models");
```
Comment (jmckenney) [inline controllers/childrenController.js]: Not suggesting a change... But controller may not be the best word for this file. This is more of an orm type of file. childrenOrm or childrenObjectMapper or ???

File: routes/api/children.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const childrenController = require('../../controllers/childrenController');
+
+module.exports = (app) =>{
+    app.post('/api/createChild',childrenController.createChild);
+    app.get("/api/findAllByChild", childrenController.findAllByChild);
+    app.get("api/findbyid", childrenController.findById);
+    app.put("/api/updatechildbudget", childrenController.updateChildBudget);
```
Comment (jmckenney) [inline routes/api/children.js]: seeing a mix of camelCase and lowercase in here. Not a big issue but always good to keep consistency.

File: routes/api/job.js [hunk 4]
```diff
@@ -1,5 +1,27 @@
-const router = require("express").Router();
+// const router = require("express").Router();
```
Comment (jmckenney) [inline routes/api/job.js]: Do your best to keep comments out of commits like this. Becomes stale very quickly.

File: controllers/childrenController.js (hunk 1 above)
Comment (saikay) [reply controllers/childrenController.js]: I was going to use destroyChild, but Josh Moore stop me from coding it that way. :|

**Pull request merged**